phpstan.neon

```diff
@@ -321,3 +321,13 @@ parameters:
                 - packages/skipper/src/Skipper/Skipper.php
 
         - '#Property Symplify\\Skipper\\ValueObject\\SkipRules\:\:\$skippedClasses \(array<string, array<string\>\|null\>\) does not accept array<string\>#'
+
+        -
+            message: '#Spread operator is not allowed#'
+            paths:
+                - packages/easy-hydrator/src/ArrayToValueObjectHydrator.php # 40
+                - packages/easy-hydrator/src/TypeCaster/ObjectTypeCaster.php # 78
+                - packages/easy-hydrator/tests/ArrayToValueObjectHydratorTest.php # 141
+                - packages/package-builder/src/Reflection/PrivatesCaller.php # 28
+                - packages/symfony-static-dumper/src/HttpFoundation/ControllerContentResolver.php # 63
+                - packages/symfony-static-dumper/src/HttpFoundation/ControllerContentResolver.php # 87
```

Comment:
Could you refactor cases to explicit? Where it can be

Reply:
I will try.

Reply:
implemented, yes, not all is possible.